department-of-veterans-affairs/vets-website
# [Check-in] Update error page for failed validation

## Description
This PR Updates the error message and styling on the error page after repeated failed validations.. This change is for both check-in and pre-check-in, and only takes effect when the telephone appointments feature flag is enabled.
Still waiting on final confirmation of the content for check-in specifically.

## Original issue(s)
department-of-veterans-affairs/va.gov-team#42351


## Testing Instructions
- Enable phone appointments feature toggle
- Input an incorrect login on any UUID for both check-in and pre-check-in apps

## Screenshots
Pre-Check-In
![Screen Shot 2022-06-16 at 1 31 51 PM](https://user-images.githubusercontent.com/5032149/174150180-6e0a2ea1-eba3-48a7-9859-0643faff7e73.png)

Check-in
![Screen Shot 2022-06-16 at 2 51 18 PM](https://user-images.githubusercontent.com/5032149/174161829-4f6f71bc-d02c-4be1-b3d7-8bfeb6057b67.png)



## Acceptance criteria

## Definition of done
